.azure/pipelines.yml

```diff
@@ -4,26 +4,59 @@ trigger:
       - master
   tags:
     include:
-      # todo: should we just include all tags? (see conditional package upload step)
-      - v0.*
+      - v*
 
-jobs:
-- template: template.yml
-  parameters:
-    name: 'Linux'
-    pool:
-      vmImage: 'ubuntu-latest'
-    packages: true
+stages:
+- stage: dev
+  displayName: Development tests
+  jobs:
+  - template: dev-template.yml
+    parameters:
+      name: 'Linux'
+      pool:
+        vmImage: 'ubuntu-latest'
+  - template: dev-template.yml
+    parameters:
+      name: 'Mac'
+      pool:
+        vmImage: 'macos-10.14'  
+  - template: dev-template.yml
+    parameters:
+      name: 'Windows'
+      pool:
+        vmImage: 'vs2017-win2016'
+      also_forked: false
 
-- template: template.yml
-  parameters:
-    name: 'Mac'
+- stage: package
+  displayName: Upload to azure feed
+  condition: and(succeeded(), startsWith(variables['build.sourceBranch'], 'refs/tags/v'))
```

Comment:
Apparently no pattern matching is possible; all tags starting with `v` will trigger.